### PR TITLE
Fix missing HoverEnter event to next Object after pointerFocusEventOb…

### PR DESCRIPTION
**Pull request by wineee**
…ject destroyed

### CODE REVIEW

File: src/server/kernel/wseat.cpp [hunk 1]
```diff
@@ -184,6 +184,10 @@ class Q_DECL_HIDDEN WSeatPrivate : public WWrapObjectPrivate
             onEventObjectDestroy = QObject::connect(eventObject, &QObject::destroyed,
                                                     q_func(), [this] {
                 doClearPointerFocus();
+                // Qt missing HoverEnter event to next Object after pointerFocusEventObject destroyed
```
Comment (zccrs) [inline src/server/kernel/wseat.cpp:187]: See frameSynchronousHoverEnabled

**Review by zccrs — CHANGES_REQUESTED** [review]

File: src/server/kernel/wseat.cpp [hunk 2]
```diff
@@ -184,6 +184,10 @@ class Q_DECL_HIDDEN WSeatPrivate : public WWrapObjectPrivate
...
+                QHoverEvent hoverEvent(QEvent::HoverEnter, QPointF(0,0), cursor->position(), cursor->position(),
+                               Qt::NoModifier);
```
Comment (zccrs) [inline src/server/kernel/wseat.cpp:189]: Why is Qt::NoModifier

File: src/server/kernel/wseat.cpp [hunk 3]
```diff
@@ -184,6 +184,10 @@ class Q_DECL_HIDDEN WSeatPrivate : public WWrapObjectPrivate
             onEventObjectDestroy = QObject::connect(eventObject, &QObject::destroyed,
                                                     q_func(), [this] {
                 doClearPointerFocus();
+                // Qt missing HoverEnter event to next Object after pointerFocusEventObject destroyed
+                QHoverEvent hoverEvent(QEvent::HoverEnter, QPointF(0,0), cursor->position(), cursor->position(),
```
Comment (zccrs) [inline src/server/kernel/wseat.cpp:188]: Why scene position is 0,0

**Review by zccrs — CHANGES_REQUESTED** [review]

**Comment by wineee** [comment]
Seems https://github.com/vioken/waylib/pull/523 fixed it

**Pull request closed without merging**